specification/ml/get_calendars/MlGetCalendarsRequest.ts

```diff
@@ -23,22 +23,29 @@ import { Id } from '@_types/common'
 import { integer } from '@_types/Numeric'
 
 /**
+ * Retrieves configuration information for calendars.
  * @rest_spec_name ml.get_calendars
  * @since 6.2.0
  * @stability stable
+ * @cluster_privileges monitor_ml
  */
 export interface Request extends RequestBase {
   path_parts: {
-    /** A string that uniquely identifies a calendar. */
+    /** A string that uniquely identifies a calendar. You can get information for multiple calendars by using a comma-separated list of ids or a wildcard expression. You can get information for all calendars by using `_all` or `*` or by omitting the calendar identifier.*/
     calendar_id?: Id
```

Comment:
Should this parameter be required if we omit `body`?

Reply:
I don't think it's required when you omit the body. For example, these work in 7.16:

> GET _ml/calendars/
> GET _ml/calendars?from=0

However, if you try to specify both a calendar ID and a query parameter, you get an error. For example:

> GET _ml/calendars/_all?from=0
... returns the following error:
> "Validation Failed: 1: Params [from, size] are incompatible with [calendar_id].